mautrix_signal/example-config.yaml

```
@@ -265,7 +265,12 @@ bridge:
         # The number of days after a puppets last activity where they are considered inactive again.
         puppet_inactivity_days: 30
         # Should the bridge block traffic when a limit has been reached
-        block_on_limit_reached: false
+        block_on_limit_reached: true
+        # The message sent to bridge admins when the bridges starts/stops being blocked
+        block_begins_notification: 'The bridge is currently blocking messages. You may need to increase your usage limits in EMS'
+        block_ends_notification: 'The bridge is no longer blocking messages'
```

I couldn't find the full stops added anywhere else.

```suggestion
        block_begins_notification: 'The bridge is currently blocking messages. You may need to increase your usage limits in EMS.'
        block_ends_notification: 'The bridge is no longer blocking messages.'
```